File: 3333/3333.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var findTheOriginalTypedStringIi = function(root) {
    if (!root) return 0;
    return 1 + Math.max(findTheOriginalTypedStringIi(root.left), findTheOriginalTypedStringIi(root.right));
};

File: 3333/3333.go
func findTheOriginalTypedStringIi(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findTheOriginalTypedStringIi(root.Left)
	r := findTheOriginalTypedStringIi(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}